core/src/nodes/node.rs

```diff
@@ -25,7 +25,7 @@ use std::fmt;
 use std::io::Error as IoError;
 use std::sync::Arc;
 
-// Implementor notes
+// Implementer notes
```

Comment:
`Implementor` is also correct (and I personally prefer it). In this particular spot perhaps `Implementation notes` could work too?

Reply:
Both are apparently correct, although implementer is more common and how I have always heard it pronounced. [ngram](https://books.google.com/ngrams/graph?content=implementor%2Cimplementer&year_start=1900&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cimplementor%3B%2Cc0%3B.t1%3B%2Cimplementer%3B%2Cc0).